web/package.json

```diff
@@ -34,13 +35,15 @@
     "@types/webpack-env": "^1.15.1",
     "@typescript-eslint/eslint-plugin": "^8.0.0",
     "@typescript-eslint/parser": "^8.0.0",
+    "@vitest/coverage-istanbul": "^2.0.5",
```

Comment:
Nit: can this go in `devDependencies`?

Reply:
indulge a digression. I have this conviction that there is no distinction between dependencies and dev dependencies in a front end non library project. like the whole thing gets built. there are no real dependencies after that. right? also by any measure i feel like most of the classification here is wrong. should typescript, and sentry cli and all the types also be moved to dev deps?

Reply:
Digression indulged. Yes seems like `devDependencies` also get installed since we don't install in the web image with the `--production` flag. Perhaps we should add that but can be deferred in a follow up PR.

Reply:
ahh but that wouldn't work. to do the build we need most of the stuff that is in dev dependencies. i think it's a distinction that makes sense in backend projects and just got carried into frontend somehow.

Reply:
but the multistage build means node modules get's left behind anyway so all good.